pkg/cmd/server/start/start_master.go

```diff
@@ -442,8 +445,8 @@ func StartAPI(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) error {
 	}
 
 	// Must start policy caching immediately
+	oc.Informers.StartCore(utilwait.NeverStop)
 	oc.RunGroupCache()
```

Comment:
> If you are going to move these in StartCore as well, open a separate issue.

Those are more controllers than caches.  I'm hoping to avoid moving them.